github/apps_marketplace.go

```diff
@@ -14,7 +14,18 @@ import (
 // methods of the GitHub API.
 //
 // GitHub API docs: https://developer.github.com/v3/apps/marketplace/
-type MarketplaceService service
+type MarketplaceService struct {
+	client *Client
+	// Stubbed controls whether endpoints that return stubbed data are used
+	// instead of production endpoints. Stubbed data is fake data that's useful
+	// for testing your GitHub Apps. Stubbed data is hard-coded and will not
+	// change based on actual subscriptions.
+	//
+	// GitHub API docs: https://developer.github.com/v3/apps/marketplace/
+	Stubbed bool
+
+	ListOptions
```

Comment:
`ListOptions` probably doesn't belong here in a service struct, does it? I'm guessing you forgot to remove it from back when we considered a `MarketplaceOptions` wrapper, yeah?

Reply:
Yurp, pushed a fix.